packages/rum/test/rum.spec.ts

```diff
@@ -452,36 +440,34 @@ describe('rum global context', () => {
   })
 
   it('should not be automatically snake cased', () => {
-    RUM.setRumGlobalContext({ fooBar: 'foo' })
+    const { server, lifeCycle, rumApi } = setupBuilder.build()
+    server.requests = []
```

Comment:
I've seen this multiple times.. The server can receive requests between its creation and the test execution? I'd be in favor of reseting the server requests in a setup builder "build task".

Reply:
it depends if you are interested or not in the events generated when starting rum.

Reply:
Ah, I see! All right.